ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/AttestationValidator.java

```diff
@@ -210,7 +224,8 @@ private InternalValidationResult singleAttestationChecks(final Attestation attes
                                 ancestorOfLMDVote ->
                                     ancestorOfLMDVote.equals(finalizedCheckpoint.getRoot()))
                             .orElse(false)) {
-                          return InternalValidationResult.REJECT;
+                          return InternalValidationResult.reject(
+                              "Attestation block root does not descent from finalized checkpoint");
```

Comment:
below we use 'does not descend', do we want that same description here?

Reply:
Done.